README.md

```diff
@@ -4579,27 +4579,33 @@ try controller.performFetch()
 
 **GRDB can encrypt your database with [SQLCipher](http://sqlcipher.net) v3.4.0.**
 
-This requires a manual installation of GRDB:
+Using CocoaPods
+```ruby
+pod 'GRDB.swift/SQLCipher'
+```
+
+Using a manual installation of GRDB:
 
 1. Clone the GRDB.swift git repository, checkout the latest tagged version, and download SQLCipher sources:
     
     ```sh
     cd [GRDB.swift directory]
     git checkout v0.107.0
     git submodule update --init SQLCipher/src
-    ````
+    ```
     
 2. Embed the `GRDB.xcodeproj` project in your own project.
 
 3. Add the `GRDBCipherOSX` or `GRDBCipheriOS` target in the **Target Dependencies** section of the **Build Phases** tab of your application target.
 
 4. Add the `GRDBCipher.framework` from the targetted platform to the **Embedded Binaries** section of the **General**  tab of your target.
 
+5. Import `GRDBCipher` module in your project's files
```

Comment:
This is not currently true with this PR: currently one has to `import GRDB`. Is it possible to have the `GRDB.swift/SQLCipher` pod define the `GRDBCipher` module? Maybe with a specific `module_name` for the `SQLCipher` subspec?